.rubocop.yml

```diff
@@ -113,6 +113,7 @@ Metrics/ModuleLength:
     - "app/models/institution_builder.rb"
 
 Metrics/BlockLength:
+  Max: 40
```

Comment:
What's going on here?

Reply:
just increasing value since default of 25 was causing lint failures